fe1-web/src/core/functions/Array.ts

```diff
@@ -1,25 +1,11 @@
 /**
- * Shuffles an array in-place
- * Copied from https://stackoverflow.com/a/2450976/2897827
+ * Creates a copy of an array and shuffles it
```

Comment:
we should probably highlight in the documentation that the shuffling is not using cryptographically safe randomness, to prevent future misuses.